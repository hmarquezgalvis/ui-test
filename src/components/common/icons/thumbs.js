import React from 'react';

import ThumbUpImage from '../../../assets/images/thumbs-up.svg';
import ThumbDownImage from '../../../assets/images/thumbs-down.svg';

const ThumbUpIcon = ({ altText = '' }) => <img src={ThumbUpImage} alt={altText} />;
const ThumbDownIcon = ({ altText = '' }) => <img src={ThumbDownImage} alt={altText} />;

export { ThumbUpIcon, ThumbDownIcon };
