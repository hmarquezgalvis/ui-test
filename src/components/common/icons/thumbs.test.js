import { render } from '@testing-library/react';
import { ThumbDownIcon, ThumbUpIcon } from './thumbs';

describe('Tests for Thumb Icons', () => {

  describe('should render and save the snapshot correctly', () => {
    it('Thumb Up Icon', () => {
      const { asFragment } = render(<ThumbUpIcon />);
      expect(asFragment()).toMatchSnapshot();
    });

    it('Thumb Down Icon', () => {
      const { asFragment } = render(<ThumbDownIcon />);
      expect(asFragment()).toMatchSnapshot();
    });

  });

});