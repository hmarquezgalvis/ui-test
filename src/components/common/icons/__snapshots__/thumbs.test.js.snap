// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Thumb Icons should render and save the snapshot correctly Thumb Down Icon 1`] = `
<DocumentFragment>
  <img
    alt=""
    src="thumbs-down.svg"
  />
</DocumentFragment>
`;

exports[`Tests for Thumb Icons should render and save the snapshot correctly Thumb Up Icon 1`] = `
<DocumentFragment>
  <img
    alt=""
    src="thumbs-up.svg"
  />
</DocumentFragment>
`;
